expr/types_test.go

```diff
@@ -704,6 +704,47 @@ func TestArrayIsCompatible(t *testing.T) {
 	}
 }
 
+func TestObjectRename(t *testing.T) {
+	cases := map[string]struct {
+		n, m     string
```

Comment:
It is a little difficult to understand what `n` and `m` variables are. Adding some comments on what `n`, `m`, and `expected` or renaming the variables would be helpful.

Reply:
OK. 88edc4b renamed the variables.